bluepyopt/ephys/locations.py

```diff
@@ -133,7 +133,7 @@ def instantiate(self, sim=None, icell=None):
     def __str__(self):
         """String representation"""
 
-        return '%s[%s]' % (self.seclist_name, self.sec_index)
+        return '%s: %s[%s]' % (self.name, self.seclist_name, self.sec_index)
```

Comment:
As far as I remember, I designed these strings, so that they could be used by higher classes to form nice sentences. I don't think that will work after this change anymore ?